src/NzbDrone.Common/Extensions/NumberExtensions.cs

```diff
@@ -26,5 +26,25 @@ public static string SizeSuffix(this long bytes)
 
             return string.Format(CultureInfo.InvariantCulture, "{0:n1} {1}", adjustedSize, SizeSuffixes[mag]);
         }
+
+        public static long Megabytes(this int megabytes)
```

Comment:
This code was just moved so it can be used for configuration. It's turning the number of megabytes into a number of bytes, not turning a number of bytes into megabytes, so the input being megabytes makes sense.

Reply:
Sorry about that. Absolutely.